server/index.js

```diff
@@ -123,6 +113,23 @@ app.get('/api/reviews/meta/:id', (req, res) => {
       res.sendStatus(404);
     });
 });
+
+app.get('/api/cart', (req, res) => {
+  axios.get(`${API_BASE_URL}/cart`, options)
+    .then((data) => res.send(data.data))
+    .catch(() => {
+      res.sendStatus(404);
+    });
+});
+
+app.post('/api/cart', (req, res) => {
+  axios.post(`${API_BASE_URL}/cart`, req.body, options)
```

Comment:
Do we need to pass the entire req.body here?

Reply:
Yes, the req.body in this case is just a single key object {sku_id: number}. It was stated in the API documentation this is how items get added to the cart database.